docs/book.json

```diff
@@ -0,0 +1,19 @@
+{
+  "plugins": ["edit-link", "prism", "-highlight", "github", "anchorjs"],
+  "pluginsConfig": {
+    "edit-link": {
+      "base": "https://github.com/kube-aws/docs/tree/master",
+      "label": "Edit This Page"
+    },
+    "github": {
+      "url": "https://github.com/kubernetes-incubator/kube-aws"
+    }
+  },
+  "links": {
+    "sidebar": {
+    }
+  },
+  "styles": {
+    "website": "styles/site.css"
+  }
+}
```

Comment:
Yes but honestly it's a low priority thing because the reason I want it is just for gitbook compatibility.
We can defer it until gitbook fixes the issue with repos under github orgs.

I'd appreciate it if you could address it though 😉